### PR TITLE
Tag SGP4 v0.1.1 [https://github.com/crbinz/SGP4.jl]

### PR DESCRIPTION
**Pull request by crbinz**
New build process

### CODE REVIEW

File: SGP4/versions/0.1.1/requires [hunk 1]
```diff
@@ -0,0 +1,4 @@
+julia 0.4
+PyCall 1.6
+Dates
+Compat
```
Comment (ararslan) [inline SGP4/versions/0.1.1/requires]: It's usually a good idea to specify a minimum version for `Compat` that includes the features you need.

**Comment by ararslan** [comment]
You can ignore the current Travis failure about the commit SHA not matching; the SHAs actually do match, so the check is wrong.

**Comment by crbinz** [comment]
I've added the Compat version as requested.

**Comment by tkelman** [comment]
`Compat.ASCIIString` was introduced a bit earlier than 0.9.0, there's a note on the Compat readme with instructions on how to find more specific versions.

**Comment by crbinz** [comment]
Thanks, that's useful.

**Pull request merged**